.changeset/ten-tools-exercise.md

```diff
@@ -0,0 +1,8 @@
+---
+'@atproto/ozone': minor
+'@atproto/bsky': minor
+'@atproto/api': minor
+'@atproto/pds': minor
```

Comment:
we definitely what the pds at least to be a `patch` bump since this intersects with our distribution verisioning & we don't want to require a hands on migration